draft-ietf-dnsop-svcb-https.md

```diff
@@ -1227,6 +1227,66 @@ HTTPS RRs, all connections will be upgraded to HTTPS, and clients will
 use HTTP/3 if they can.  Parameters are "bound" to each server pool, so
 each server pool can have its own protocol, ECH configuration, etc.
 
+## Multi-CDN
+
+The HTTPS RR is intended to support HTTPS services operated by
+multiple independent entities, such as different Content Delivery
+Networks (CDNs) or different hosting providers.  This includes
+the case where a service is migrated from one operator to another,
+as well as the case where the service is multiplexed between
+multiple operators for performance, redundancy, etc.
+
+This example shows such a configuration, with www.customer.example
+having different DNS responses to different queries, either over time
+or due to logic within the authoritative DNS server:
+
+     $ORIGIN customer.example.  ; A Multi-CDN customer domain
+     www 900 IN CNAME cdn1.svc1.example.      ; For some subset of DNS responses
```

Comment:
This is illegal. RFC 1034 (and addition RFCs clarifying this) restrict the owner to having a single CNAME RR (and prohibit any other RRTYPE with the same owner as a CNAME).
Specifically, you cannot have two CNAME records with the same owner name (in this case "www".)

Reply:
Yes.  The point is that only one of these exists in the RRset at any one time.  I'll try and clarify this.

Reply:
Is the updated text better?

Reply:
If we need to be clearer, it might be worth moving the explanatory text from comments to regular prose, and having two distinct code blocks.  Also, this line and 1250 are too long to fit on a single line in a code block.